packages/base-driver/test/basedriver/driver-tests.js

```diff
@@ -75,12 +78,12 @@ function baseDriverUnitTests (DriverClass, defaultCaps = {}) {
       let sessions = await d.getSessions();
 
       sessions.length.should.equal(1);
-      sessions[0].should.eql({
-        id: d.sessionId,
-        capabilities: {
-          'deviceName': 'Commodore 64',
-          'platformName': 'Fake'
-        }
+      sessions[0].should.include({
+        id: d.sessionId
+      });
+      sessions[0].capabilities.should.include({
+        'deviceName': 'Commodore 64',
```

Comment:
I assume there is no need to quote key names

Reply:
no, but we don't auto-format the code and eslint doesn't seem to have a rule enabled about it, so ... we should address that

Reply:
i thought we had an eslint rule. i guess not. we should add a rule

Reply:
even if there's not a rule, let's unquote these key names as part of this review